techyteam/Ifman
# #167008186 build landing page

#### What does this PR do?
Setup project and build the landing page
#### Description of Task to be completed?
- setup UI folders
- build landing page 
#### How should this be manually tested?
Clone this branch.
navigate to https://github.com/misanj/Ifman/ to see the background page
#### What are the relevant pivotal tracker stories?
#167008186
